lib/ex_unit/lib/ex_unit/assertions.ex

```diff
@@ -392,8 +392,11 @@ defmodule ExUnit.Assertions do
       Regex.regex?(message) -> Exception.message(error) =~ message
     end
 
-    msg = "Wrong message for #{inspect exception}. " <>
-          "Expected #{inspect message}, got #{inspect Exception.message(error)}"
+    msg = "Wrong message for #{inspect exception}\n" <>
```

Comment:
@eksperimental I initially put it there but removed it as there's a newline now. Other error messages don't have a trailing period. Wdyt?